cmd/crates/soroban-spec-typescript/src/project_template/src/method-options.ts

```diff
@@ -15,7 +15,23 @@ export interface Wallet {
   }) => Promise<XDR_BASE64>,
 }
 
-export type Options<R extends ResponseTypes> = {
+export type ClassOptions = {
+  contractId: string
+  networkPassphrase: string
+  rpcUrl: string
+  /**
+   * A Wallet interface, such as Freighter, that has the methods `isConnected`, `isAllowed`, `getUserInfo`, and `signTransaction`. If not provided, will attempt to import and use Freighter. Example:
+   *
+   * ```ts
```

Comment:
```suggestion
   * @example
```
I _think_ this is the right way to render it? If not, you need the closing ``` anyway

Reply:
Thanks, [looks like](https://typedoc.org/tags/example/) the best way is to use both `@example` and the backticks together